react-native-libraries.json

```
@@ -16816,5 +16816,20 @@
     "web": true,
     "expoGo": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/10play/10tap-editor",
+    "npmPkg": "@10play/tentap-editor",
+    "examples": [
+      "https://github.com/10play/10tap-editor/tree/main/example",
+      "https://github.com/10play/10TapAdvancedExample",
+      "https://github.com/10play/10tap-web-example"
+    ],
+    "images": ["https://github.com/10play/10tap-editor/blob/main/website/static/img/cover.webp"],
```

```suggestion
    "images": ["https://raw.githubusercontent.com/10play/10tap-editor/refs/heads/main/website/static/img/cover.webp"],
```

Update image URL to point to raw asset.